### ISSUE TITLE
HTML Semantics 

### ISSUE THREAD

**Issue by Andrey-1992** [Andrey-1992/suspicious-ip-ui#1]
As a developer, I should be able to constructor a responsive and accessible app based on HTML semantics tags.
